app/settings.py

```diff
@@ -1,28 +1,48 @@
+import os
+
 # These paths are mounted into the docker container by docker-entrypoint.sh
-WATCH_FOLDER = "/mount/watch"
-MASTER_FOLDER = "/mount/master"
-ACCESS_FOLDER = "/mount/access"
-OUTPUT_FOLDER = "/mount/output"
+WATCH_FOLDER = os.getenv("WATCH_FOLDER", "/mount/watch/")
+MASTER_FOLDER = os.getenv("MASTER_FOLDER", "/mount/master/")
+ACCESS_FOLDER = os.getenv("ACCESS_FOLDER", "/mount/access/")
+WEB_FOLDER = os.getenv("WEB_FOLDER", "/mount/web/")
+OUTPUT_FOLDER = os.getenv("OUTPUT_FOLDER", "/mount/output/")
 
-FFMPEG_DESTINATION_EXT = ".mp4"
-FFMPEG_ARGS = [
+ACCESS_FFMPEG_DESTINATION_EXT = ".mp4"
+ACCESS_FFMPEG_ARGS = [
     '-loglevel', 'panic',
     '-stats',
     '-hide_banner',
     '-pix_fmt', 'yuv420p',  # colour format compatible with quicktime
     '-c:v', 'libx264',
-    '-preset', 'medium',
+    '-preset', 'veryslow',
```

Comment:
We may need to have a separate ticket to do some comparisons of time and file size for different presets. In my local machine using docker converting a 15Mb file took around 3min. If the compression time is linear with larger file sizes, then a 50Gb file (not uncommon in collections videos) would take around 7 days. This of course depends where the transcoder will run.